pkg/kubectl/describe.go

```diff
@@ -766,8 +766,8 @@ func (d *PersistentVolumeClaimDescriber) Describe(namespace, name string) (strin
 	})
 }
 
-// DescribeContainers is exported for consumers in other API groups that have container templates
-func DescribeContainers(containers []api.Container, containerStatuses []api.ContainerStatus, resolverFn EnvVarResolverFunc, out io.Writer) {
+// describeContainers is exported for consumers in other API groups that have container templates
+func describeContainers(containers []api.Container, containerStatuses []api.ContainerStatus, resolverFn EnvVarResolverFunc, out io.Writer) {
```

Comment:
@smarterclayton IIRC you add this function at begin, is that ok to make this function non pubic?